build.sh

```diff
@@ -147,6 +152,6 @@ fi
 if (( NUMARGS == 0 )) || hasArg kvikio; then
     echo "building kvikio..."
     cd ${REPODIR}/python
-    SKBUILD_CMAKE_ARGS="-DCMAKE_PREFIX_PATH=${INSTALL_PREFIX};-DCMAKE_LIBRARY_PATH=${LIBKVIKIO_BUILD_DIR};${EXTRA_CMAKE_ARGS}" \
+    SKBUILD_CMAKE_ARGS="-DCMAKE_PREFIX_PATH=${INSTALL_PREFIX};-DCMAKE_LIBRARY_PATH=${LIBKVIKIO_BUILD_DIR};-DFIND_KVIKIO_CPP=ON;${EXTRA_CMAKE_ARGS}" \
```

Comment:
Is this necessary? We already added to EXTRA_CMAKE_ARGS above. cuDF doesn’t do this: https://github.com/rapidsai/cudf/blob/b5f6aa59cd9d2ebb238f9f249b305d1883169332/build.sh#L340

Reply:
Oh yeah, that was left over from an earlier iteration and I forgot to remove it. Fixed.